### PR TITLE
lot of cleanup and making it run locally

### PR DESCRIPTION
**Pull request by grosser**
@jespr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,15 @@
...
 
-`minitest_converter [path-to-file]`
+`minitest_converter [path-to-file-or-folder]`
 
 It converts `should` to `it`
 `context` to `describe`
 `setup` to `before`
-
-
-## Contributing
```
Comment (grosser) [inline README.md:28]: most likely this is not the first project someone will contribute too :)

**Comment by jespr** [comment]
:boom: :+1:

File: README.md (hunk 1 above)
Comment (jespr) [inline README.md:28]: whaaa! ;)

**Comment by grosser** [comment]
You'll have to hit merge :)

**Comment by jespr** [comment]
I thought i had done that :(

**Pull request merged**